datajoint/blob.py

```diff
@@ -73,7 +73,7 @@ def __init__(self, squeeze=False):
 
     def set_dj0(self):
         if not config.get('enable_python_native_blobs'):
-            raise DataJointError('v0.12+ python native blobs disabled. see also: https://github.com/datajoint/datajoint-python/blob/master/README.md')
+            raise DataJointError('v0.12+ python native blobs disabled. see also: https://github.com/datajoint/datajoint-python#python-native-blobs')
```

Comment:
While this is a more direct link, it also disambiguate the referral to the underlying README.md in the sources. Not sure what is better.

Reply:
the link is meant t point to the specific section